modules/learn/pages/buckets-memory-and-storage/storage.adoc

```diff
@@ -65,10 +65,34 @@ Each tombstone includes the item's key and metadata.
 Tombstones are maintained in order to provide eventual consistency both between nodes and between clusters.
 
 The _Metadata Purge Interval_ establishes the frequency with which Couchbase Server _purges_ itself of tombstones: which means, removes them fully and finally.
-The Metadata Purge Interval setting runs as part of auto-compaction.
+The Metadata Purge Interval setting runs as part of auto-compaction (see xref:learn:buckets-memory-and-storage/storage.adoc#append-only-writes-and-auto-compaction[Append-Only Writes and Auto-Compaction], below).
 
 For more information, see xref:buckets-memory-and-storage/expiration.adoc#post-expiration-purging[Post-Expiration Purging], in xref:buckets-memory-and-storage/expiration.adoc[Expiration].
 
+[#disk-paths]
+== Disk Paths
+
+At node-initialization, Couchbase Server allows up to four custom paths to be established, for the saving of data to the filesystem: the Data Service, the Index Service, the Analytics Service, and the Eventing Service can thereby each be assigned their own, separate data path. Note that these paths are node-specific: consequently, the data for any of these services may occupy a different filesystem-location, on each node.
```

Comment:
suggest..
At node-initialization, Couchbase Server allows up to four custom paths to be established, for the saving of data to the filesystem: the Data Service, the Index Service, the Analytics Service, and the Eventing Service.  These four paths are node-specific: consequently, the data for any of these services may occupy a different filesystem-location, on each node.